spec/pagseguro/payment_status_spec.rb

```diff
@@ -1,26 +1,35 @@
+# -*- encoding: utf-8 -*-
 require "spec_helper"
 
-shared_examples_for "payment status mapping" do |id, status|
+shared_examples_for "payment status mapping" do |id, status, description|
+  before :all do
```

Comment:
@Lrcezimbra you can just use `let(:payment_status) {  PagSeguro::PaymentStatus.new(id) }` 

And you access this variable using `payment_status` instead of being an instance variable (@payment_status)